### PR TITLE
fix system test for edit,delete and purge address

### PR DESCRIPTION
**Pull request by drcoolsanjeev**
### Type of change

- Bugfix
- Enhancement / new feature

### Description
 Fix system tests

### CODE REVIEW

**Comment by drcoolsanjeev** [comment]
@enmasse-ci run tests testcase=isolated.web.FirefoxConsoleTest.*

File: systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java [hunk 1]
```diff
@@ -136,15 +136,15 @@ public WebElement getActionDropDown() {
     }
 
     public WebElement getEditMenuItem() {
-        return getActionDropDown().findElement(By.xpath("//a[contains(text(), 'Edit')]"));
+        return getActionDropDown().findElement(By.xpath("//*[@id='edit-address']/div"));
```
Comment (kornys) [inline systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java:139]: use `findElement(By.id('your-id'))`

File: systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java [hunk 2]
```diff
@@ -136,15 +136,15 @@ public WebElement getActionDropDown() {
...
     }
 
     public WebElement getDeleteMenuItem() {
-        return getActionDropDown().findElement(By.xpath("//a[contains(text(), 'Delete')]"));
+        return getActionDropDown().findElement(By.xpath("//*[@id='delete-address']/div"));
```
Comment (kornys) [inline systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java:143]: same as above

File: systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java [hunk 3]
```diff
@@ -136,15 +136,15 @@ public WebElement getActionDropDown() {
...
     }
 
     public WebElement getPurgeMenuItem() {
-        return getActionDropDown().findElement(By.xpath("//a[contains(text(), 'Purge')])]"));
+        return getActionDropDown().findElement(By.xpath("//*[@id='purge-address']/div"));
```
Comment (kornys) [inline systemtests/src/main/java/io/enmasse/systemtest/selenium/resources/AddressWebItem.java:147]: same as above

**Comment by drcoolsanjeev** [comment]
@enmasse-ci run tests testcase=*.web.FirefoxConsoleTest

**Comment by kornys** [comment]
@drcoolsanjeev I don't understand why it was merged into some branch even if tests failed? Do you plan to fix all tests in console-ui-iot branch?

**Pull request merged**